aten/src/ATen/test/broadcast_test.cpp

```diff
@@ -22,8 +22,7 @@ void TestOut2Basic(DeprecatedTypeProperties& T) {
 
 // with scalar
 void TestOut2WithScalar(DeprecatedTypeProperties& T) {
-  auto aScalar = ones({1}, T);
-  aScalar.unsafeGetTensorImpl()->maybe_zero_dim(true);
+  auto aScalar = ones({}, T);
```

Comment:
What was the reason to do tests like this? why not create 0-dim tensors directly?

Reply:
That's a good question :).  I can't remember if it was because I was trying to test `maybe_zero_dim` directly or these tests were added before zero-dimensional tensors were exposed publicly so this was the only API.

Reply:
Ok !